sys/parsoid.js

```diff
@@ -440,25 +441,25 @@ PSP.getFormat = function(format, restbase, req) {
         }
     }
 
+    if (!self._okayToRerender(req)) {
```

Comment:
I see, missed that. I still think it would be more obvious to avoid splitting the no-cache logic, but it'll work as-is.